src/Operators/integrals.jl

```diff
@@ -86,6 +88,19 @@ end
 Sets `ᶠ∫field```(z) = \\int_0^z\\,```ᶜfield```(z')\\,dz'``. The input `ᶜfield`
 must lie on a cell-center space, and the output `ᶠ∫field` must lie on the
 corresponding cell-face space.
+
+    column_integral_indefinite!(
+        f::Function,
+        ᶠ∫field::Fields.ColumnField,
+        ϕ₀ = 0,
+        average = (ϕ⁻, ϕ⁺) -> (ϕ⁻ + ϕ⁺) / 2,
```

Comment:
I think it would be good for the docstring to explain what `average` is and how it is used.

Also, explicitly mentioning the order of accuracy of the method would be useful, I think.

Reply:
Yeah, good point. I'll update that

Reply:
We should probably do that for all of our other operators, too..